frontend/src/scenes/surveys/EditSurveyNew.tsx

```diff
@@ -0,0 +1,969 @@
+                                                                                                        )
+                                                                                                    }}
+                                                                                                />
+                                                                                                <LemonButton
+                                                                                                    icon={
+                                                                                                        <IconDelete />
+                                                                                                    }
+                                                                                                    size="small"
+                                                                                                    status="muted"
+                                                                                                    noPadding
+                                                                                                    onClick={() => {
+                                                                                                        const newChoices =
+                                                                                                            [...value]
+                                                                                                        newChoices.splice(
+                                                                                                            index,
+                                                                                                            1
+                                                                                                        )
+                                                                                                        onChange(
+                                                                                                            newChoices
+                                                                                                        )
+                                                                                                    }}
+                                                                                                />
+                                                                                            </div>
+                                                                                        )
+                                                                                    )}
+                                                                                    <div className="w-fit">
+                                                                                        {(value || []).length < 6 && (
```

Comment:
Curious why 6 here, do you know @neilkakkar @jurajmajerik or @smallbrownbike ?
Later this was relaxed for APIs at least https://github.com/PostHog/posthog/pull/26224/